auth/src/lib.rs

```diff
@@ -98,26 +99,23 @@ impl Args for Options {
                 active,
                 stdin,
                 name,
-                peer_id,
+                node_id,
             },
             vec![],
         ))
     }
 }
 
 pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
-    let profiles = match profile::list() {
-        Ok(profiles) => profiles,
-        _ => vec![],
-    };
+    let profile = Profile::load();
 
-    if options.init || profiles.is_empty() {
-        if options.peer_id.is_some() {
-            anyhow::bail!("you may not specify a peer id when initializing a new identity");
+    if options.init || profile.is_err() {
+        if options.node_id.is_some() {
+            anyhow::bail!("you may not specify a node id when initializing a new identity");
         }
         init(options)
     } else {
-        authenticate(&profiles, options, ctx)
+        authenticate(&profile.unwrap(), options, ctx)
     }
```

Comment:
We can get rid of `--init` and simplify this whole function, since you can't init a profile if you already have one!

Reply:
Also see `radicle-tools/rad-auth` for how to check that the profile doesn't exist.